### PR TITLE
New benchmark for fadd lvl2

### PR DESCRIPTION
**Pull request by ledentp**
This PR proposes an implementation of a benchmark for the matrix/matrix addition.

### CODE REVIEW

**Comment by ledentp** [comment]
Commit e6ce3e3 should not have been in this branch but there is no conflict because it has been repaired in a branch whose purpose was to fix the memory leak in benchmark-fsyrk.C .

File: benchmarks/benchmark-fadd-lvl2.C [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  Element alpha;
+  F.init(alpha, a);
+
+	
+  FFLAS::Timer chrono;
+  double time=0.0;
+	
+  for (size_t i=0;i<=iter;++i){
+	A = fflas_new(F, rows, cols);
+    size_t lda = cols;
```
Comment (ClementPernet) [inline benchmarks/benchmark-fadd-lvl2.C]: Indentation (4 spaces as the first line after the "for")

File: benchmarks/benchmark-fadd-lvl2.C [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  int    q         = 131071;
+  size_t rows      = 1000;
+  size_t cols      = 1000;
+  double a         = 1.0;
+  size_t threshold = 64;
+  std::string file = "";
+  
+  Argument as[] = {
+    { 'q', "-q Q", "Set the field characteristic (-1 for random).",         TYPE_INT , &q },
+	{ 'm', "-m M", "Set the row dimension of the matrix C.",                TYPE_INT , &rows },
```
Comment (ClementPernet) [inline benchmarks/benchmark-fadd-lvl2.C]: indentation

File: benchmarks/benchmark-fadd-lvl2.C [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+
+using namespace std;
+using namespace FFLAS;
+using namespace FFPACK;
+int main(int argc, char** argv) {
+  
+  size_t iter      = 3;
+  int    q         = 131071;
+  size_t rows      = 1000;
```
Comment (ClementPernet) [inline benchmarks/benchmark-fadd-lvl2.C]: you should use larger default values for rows and cols (5000). This is only a n^2 algorithm

File: benchmarks/benchmark-fadd-lvl2.C [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+ * ========LICENCE========
+ */
+
+#include "fflas-ffpack/fflas-ffpack-config.h"
+#include <iostream>
+#include <givaro/modular.h>
+
+#include "fflas-ffpack/fflas-ffpack.h"
+#include "fflas-ffpack/utils/timer.h"
+#include "fflas-ffpack/utils/Matio.h"
```
Comment (ledentp) [inline benchmarks/benchmark-fadd-lvl2.C]: Your comment seems to be incomplete. What should be used ?

Comment (ClementPernet) [inline benchmarks/benchmark-fadd-lvl2.C]: Sorry: 
... one should use `fflas_io.h` instead.

File: benchmarks/benchmark-fadd-lvl2.C (hunk 1 above)
Comment (ledentp) [inline benchmarks/benchmark-fadd-lvl2.C]: On my computer both indentations were good (4spaces). I'll try submitting (pushing) again and we'll see what happens.

Comment (ClementPernet) [inline benchmarks/benchmark-fadd-lvl2.C]: there is probably a TAB in one line and 4 spaces in the other

**Comment by ledentp** [comment]
I've commited the changes that were requested.

**Comment by ClementPernet** [comment]
It seems that you still did not remove the `#include "Matio.h"`
Also, you have added the binary `benchmark-fadd-lvl2`. Please remove it (`git rm`)

**Comment by ledentp** [comment]
Oh sorry

**Pull request merged**